modules/Sources/Features/SecurityWarning/SecurityWarningStore.swift

```diff
@@ -18,43 +18,37 @@ import Generated
 import Utils
 import SwiftUI
 
-public typealias SecurityWarningStore = Store<SecurityWarningReducer.State, SecurityWarningReducer.Action>
-public typealias SecurityWarningViewStore = ViewStore<SecurityWarningReducer.State, SecurityWarningReducer.Action>
-
-public struct SecurityWarningReducer: Reducer {
+@Reducer
+public struct SecurityWarning {
+    @ObservableState
     public struct State: Equatable {
-        public enum Destination: Equatable, CaseIterable {
-            case createNewWallet
-        }
-
-        @PresentationState public var alert: AlertState<Action>?
+        @Presents public var alert: AlertState<Action>?
         public var appBuild = ""
         public var appVersion = ""
-        public var destination: Destination?
-        @BindingState public var isAcknowledged: Bool = false
+        public var isAcknowledged: Bool = false
+        public var recoveryPhraseDisplayViewBinding: Bool = false
```

Comment:
Document when this is true, since it's not clear from the name (or use a more obvious name).

Reply:
Done